src/DynamoCoreWpf/ViewModels/Watch3D/DynamoEffectsManager.cs

```diff
@@ -0,0 +1,76 @@
+﻿using HelixToolkit.Wpf.SharpDX;
+using SharpDX.Direct3D11;
+using SharpDX.DXGI;
+
+namespace Dynamo.Wpf.ViewModels.Watch3D
+{
+    public class DynamoEffectsManager : DefaultEffectsManager
```

Comment:
This file needs careful documentation. Other developers have no idea what an `EffectsManager` is.